MapboxDirections/MBRoadClass.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+
+@objc(MBRoadClass)
+public enum RoadClass: Int, CustomStringConvertible {
+    case toll
+    
+    case restricted
+    
+    case highway
+    
+    case ferry
+    
+    public init?(description: String) {
+        let level: RoadClass
+        switch description {
+        case "toll":
+            level = .toll
+        case "restricted":
+            level = .restricted
+        case "highway":
```

Comment:
@1ec5 I've been told by @TheMarex it will be `toll`, `restricted`,`highway` and `ferry`. 

@TheMarex is this the case?